Add laser clustering example.

Paulooh007 opened the pull request:
A tutorial notebook that demonstrates multilingual embedding clustering.

avidale commented on tasks/clustering/README.md:
```diff
@@ -0,0 +1,13 @@
+
+[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/Paulooh007/LASER/blob/laser-clustering/tasks/clustering/LaserClusteringExample.ipynb)
+
+
```
Apart from opening the notebook in Colab and running all the cells, the user might want just to take a look at the resulting notebook. 

I can open one in https://github.com/Paulooh007/LASER/blob/laser-clustering/tasks/clustering/LaserClusteringExample.ipynb and it looks good, but it has no images. 

Can you embed the images directly into the notebook? I do not know if plotly can do so, but matplotlib surely can.

Paulooh007 replied:
I was able to fix this in my recent commit by using `fig.show("png")`, so it renders well on GitHub. Additionally, I added comments in case the user wants to enable interactive plots, where they can hover over the points to see more details.

heffernankevin commented:
This is really nice! A small nit: at the end you're plotting a random selected cluster, but it looks like you're adding a label "trace 10" under the "Languages" legend, and the title is "Cluster 2". Maybe something simple to try is changing the title to: "Sentences in Cluster 2 (highlighted in green)" and remove "trace 10" from the legend.

Paulooh007 commented:
> This is really nice! A small nit: at the end you're plotting a random selected cluster, but it looks like you're adding a label "trace 10" under the "Languages" legend, and the title is "Cluster 2". Maybe something simple to try is changing the title to: "Sentences in Cluster 2 (highlighted in green)" and remove "trace 10" from the legend.

Hi @heffernankevin ,
I couldn't remove the trace without removing the whole legend, So I renamed it to "Selected Cluster". Not sure if that's ok.
I've also renamed the title. Let me know what you think.
